### ISSUE TITLE
Cannot build

### ISSUE THREAD

**Issue by XiaoWang-Github** [kyuepublic/ptychopy#6]
After running this command to build your program: python setup.py build -v
I get the following error message. Seems that your program doesn't support cuda 11.3 ?

running build
running build_ext
building 'ptychopy' extension
Warning: Can't read registry to find the necessary compiler setting
Make sure that Python modules winreg, win32api or win32con are installed.
C compiler: g++ -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC

compile options: '-I/home/xf9/miniconda3/envs/epie/lib/python3.6/site-packages/numpy/core/include -I['/home/xf9/miniconda3/envs/epie/lib/python3.6/site-packages/numpy/core/include'] -I/usr/local/cuda-11.3/include -I/usr/local/cuda-11.3/samples/common/inc -I/home/xf9/miniconda3/envs/epie/include -I./src -I/home/xf9/miniconda3/envs/epie/include/python3.6m -c'
extra options: 'nvcc gcc g++'
g++: ./src/CartesianScanMesh.cpp
g++: ./src/CudaSmartPtr.cpp
nvcc: ./src/utilitiesKernels.cu
nvcc: ./src/ScanMeshKernels.cu
nvcc: ./src/ProbeKernels.cu
g++: ./src/Cuda2DArray.cpp
nvcc: ./src/SampleKernels.cu
nvcc: ./src/DiffractionsKernels.cu
nvcc fatal   : Unsupported gpu architecture 'compute_113'
nvcc fatal   : Unsupported gpu architecture 'compute_113'
nvcc fatal   : Unsupported gpu architecture 'compute_113'
nvcc fatal   : Unsupported gpu architecture 'compute_113'
g++: ./src/IPtychoScanMesh.cpp
nvcc fatal   : Unsupported gpu architecture 'compute_113'
g++: ./src/DiffractionLoader.cpp
g++: ./src/ThreadPool.cpp
g++: ./src/ListScanMesh.cpp
g++: ./src/Sample.cpp

**Comment by kyuepublic** [comment]
Yes, I got same issue with compiling with CUDA 11, I do fixed with a trick for using only C++ binary. If you check src/CMakeLists.txt, the second line that I comment out maybe give you some hint so that it might be compiled with CUDA 11. But for faster installation, CUDA version under 10 is better. And I didnt find too much performance difference.

**Comment by XiaoWang-Github** [comment]
I am using a public computer and cannot install whatever CUDA version I like. I only have access to cuda 11.0, 11.2, and 11.3 .  So for me, is there any way that I can use your program?

**Comment by kyuepublic** [comment]
what is your gcc version? the output of gcc --version?

**Comment by paidaxinbao** [comment]
Hello!
I'm having this problem with the installation process following your README file.

My device is python 3.6, CUDA 10.2, gcc and g++ are both version 4.8.5, and after running . /install.sh, I got an error halfway through the 
installation:
 
error: Command "g++ -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/home/pxy/miniconda3/envs/ptychopy/lib/ python3.6/site-packages/numpy/core/include -I['/home/pxy/miniconda3/envs/ptychopy/lib/python3.6/site-packages/numpy/core/include'] - I/usr/local/cuda-10.2/include -I/usr/local/cuda-10.2/samples/common/inc -I/home/pxy/miniconda3/envs/ptychopy/include -I./src -I/home/ pxy/miniconda3/envs/ptychopy/include/python3.6m -c . /src/FileManager.cpp -o build/temp.linux-x86_64-3.6/src/FileManager.o -DEPIE_HDF5 -fpic -DHAVE_HDF5" failed with exit status 1

I don't know what the problem is, so I'm asking for your help!Thanks!
